aten/src/ATen/native/nested/NestedTensorUtils.h

```diff
@@ -161,17 +176,18 @@ inline std::vector<IntArrayRef> NestedTensor_get_strides(
 inline void check_numel_equals_buffer_size(const at::Tensor& self) {
   auto self_impl = get_nested_tensor_impl(self);
   TORCH_CHECK(
-      self.numel() == self_impl -> get_buffer_size(),
+      self.numel() == self_impl->get_buffer_size(),
```

Comment:
I think we can remove this this function since we required get_buffer to be contiguous

Reply:
We can make that a separate PR